analogwp-templates.php

```diff
@@ -126,19 +126,44 @@ function analog_fail_load() {
 	if ( ! function_exists( 'get_plugins' ) ) {
 		require_once ABSPATH . 'wp-admin/includes/plugin.php';
 	}
-	$file_path         = 'elementor/elementor.php';
-	$installed_plugins = get_plugins();
-	$elementor         = isset( $installed_plugins[ $file_path ] );
+	$file_path = 'elementor/elementor.php';
 
-	if ( $elementor ) {
+	if ( version_compare( get_bloginfo( 'version' ), '5.5a', 'gt' ) ) {
+		$request  = new WP_REST_Request( 'GET', '/wp/v2/plugins/elementor/elementor' );
+		$response = rest_do_request( $request );
+
+		if ( $response->is_error() ) {
+			wp_die( '<p>' . esc_html__( 'An error occurred while checking Elementor is Installed/Activated', 'ang' ) . '</p>' );
+		}
+
+		$server = rest_get_server();
+		$data   = $server->response_to_data( $response, false );
+
+		if ( ! empty( $data['status'] ) && 'inactive' === $data['status'] ) {
+			$is_not_activated = true;
+		} elseif ( ! empty( $data['data']['status'] ) && 404 === $data['data']['status'] ) {
+			$is_not_installed = true;
+		}
+	} else {
+		$installed_plugins = get_plugins();
+		$elementor         = isset( $installed_plugins[ $file_path ] );
+
+		if ( $elementor ) {
+			$is_not_activated = true;
+		} elseif ( ! $elementor ) {
+			$is_not_installed = true;
+		}
```

Comment:
since you're checking against true/false values, the latter doesn't need to be elseif, simply else should do it?

Reply:
Changed from `elseif` to `else` statement